### DISCUSSION TITLE
Will the new google identity service library included in the version compatible with angular 8

### DISCUSSION THREAD

**Discussion by InuriIlangakoon** [abacritt/angularx-social-login#689] [Q&A]
Im using this is a project with angular 8. Im using the "angularx-social-login": "2.2.1", When the google deprecates the [Google Sign-In platform library](https://developers.google.com/identity/sign-in/web/sign-in) can i still use this library without any issue

**Comment by pranavwebmobi** [comment]
I also using the same and not working as well. any solutions for this?

**Comment by Heatmanofurioso** [comment]
I don't plan on maintaining that version anymore.
If you wish to do a PR with this change to a compatibility branch, I don't mind publishing an Angular V8 compatible version with that update to npm
